src/pkg/backup/details/groups.go

```diff
@@ -51,9 +59,20 @@ func (i *GroupsInfo) UpdateParentPath(newLocPath *path.Builder) {
 }
 
 func (i *GroupsInfo) uniqueLocation(baseLoc *path.Builder) (*uniqueLoc, error) {
-	return nil, clues.New("not yet implemented")
+	if len(i.DriveID) == 0 {
```

Comment:
not all groups are guaranteed to have a drive ID.  This should accommodate both variations.